subprojects/code-quality/src/integTest/groovy/org/gradle/api/plugins/quality/checkstyle/CheckstylePluginVersionIntegrationTest.groovy

```diff
@@ -404,6 +477,10 @@ class CheckstylePluginVersionIntegrationTest extends MultiVersionIntegrationSpec
         result.hasErrorOutput("[ant:checkstyle] [WARN]") || result.hasErrorOutput("warning: Name 'class1' must match pattern")
     }
 
+    private static isSarifSupported(String version) {
+        return VersionNumber.parse(version) >= VersionNumber.parse("10.3.3")
```

Comment:
:x: parent class already has a `versionNumber` member, no need to parse again here

Reply:
updated